legal-api/src/legal_api/models/filing.py

```diff
@@ -106,7 +118,11 @@ class Source(Enum):
                     'BEN': 'AMALR',
                     'BC': 'AMALR',
                     'ULC': 'AMALR',
-                    'CC': 'AMALR'
+                    'CC': 'AMALR',
+                    'CBEN': 'AMALR',
+                    'C': 'AMALR',
+                    'CUL': 'AMALR',
+                    'CCC': 'AMALR'
```

Comment:
this is not required as an amalgamationApplication can never end up in these legal types

Reply:
I looked into this a bit, I think this comes into play when getting the fee codes. I tested by removing this and certain tests fail `test_get_allowed_actions`. 
![image](https://github.com/bcgov/lear/assets/116035339/5def50d0-b5c8-42c3-8445-c2bd13549fef)